.github/workflows/helmsman_external_destroy_external.yml

```
@@ -1,60 +1,330 @@
+      RESIDENT_CAPTCHA_SITE_KEY: ${{ secrets.RESIDENT_CAPTCHA_SITE_KEY }}
+      RESIDENT_CAPTCHA_SECRET_KEY: ${{ secrets.RESIDENT_CAPTCHA_SECRET_KEY }}
+      CLUSTER_ID: ${{ github.event.inputs.clusterid || vars.CLUSTER_ID }}
+    steps:
+      - name: Show matrix values
+        run: |
+          echo "Processing ${{ matrix.dsf_files }}"
+          echo "Using WireGuard config: ${{ matrix.wg_conf }}"
+          pwd
+
+      - name: Checkout repository
+        uses: actions/checkout@v2
```

actionlint (1.7.11)

[error] the runner of "actions/checkout@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)